### PR TITLE
chore: set max SDK version to <3.0.0

### LINKED ISSUE #5: Bump to support Dart2

### PR DESCRIPTION
**Pull request by chalin**
Fixes #5

### CODE REVIEW

**Review by kevmoo — APPROVED** [review]
Will wait for green travis before I merge

**Review by kevmoo — CHANGES_REQUESTED** [review]
Need to fix the deprecated SDK constants.

Since we're not bringing in dart2_constant into google3, please bump the min SDK to 2.0.0-dev.54.0 and drop stable from travis

**Comment by kevmoo** [comment]
Can also cleanup deprecated test bits, too! https://travis-ci.org/dart-lang/multi_server_socket/jobs/405956534#L524

Thanks

File: pubspec.yaml
Comment (kevmoo) [inline pubspec.yaml]: Might as well make this `^1.3.0`

**Review by kevmoo — APPROVED** [review]

File: pubspec.yaml
Comment (chalin) [reply pubspec.yaml]: Ok, that works for me.

**Comment by chalin** [comment]
- [x] bump the min SDK to 2.0.0-dev.54.0
- [x] drop stable from travis
- [x] cleanup deprecated test bits

**Pull request merged**
